frontend/src/Editor/Editor.jsx

```diff
@@ -626,7 +626,9 @@ const EditorComponent = (props) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
-  const handleRunQuery = (queryId, queryName) => runQuery(getEditorRef(), queryId, queryName);
+  const handleRunQuery = (queryId, queryName, ...additionalArgs) => {
```

Comment:
a better approach here is to pass additionalArgs as an object.
Reason: your don't have to worry about order and setting value to null if one param in between is not requied